support/ml-advice/index.rst

```diff
@@ -14,8 +14,187 @@
+- AI models require access to relatively significant computational resources -- i.e., graphical processing units (GPUs).
+- AI requires a relatively high amount of data for training the algorithm, while very little is necessary for a CS approach.
+- CS volunteers are typically more skilled in cases where it is difficult to discern between two classes.
+
+
+
+When is each approach most optimal?
+===================================
+
+We delineate circumstances and characteristics of problems that suggest that a particular method (AI or CS) will be more beneficial for a project.
+
+
+You probably want to use AI if ...
+----------------------------------
+- There is a lot of data (e.g., at least 1,000 objects/images per class); this can be simulated or real observed data.
+- You can simulate the data for objects with reasonable fidelity -- get most of the details.
+- The morphological features that indicate a particular class are more obvious than not to the human eye.
+- The classes are already clearly identified and well understood: you know what you're looking for.
+- No classes are well-identified: anomaly detection, like with principal component analysis (PCA).
+- You are seeking to classify many millions of objects.
+
+
+You probably want to use CS if ...
+----------------------------------
+- There is not a lot of data (e.g., only a few objects/images per class); this can be simulated or real observed data.
+- The morphological features that indicate a particular class are difficult to distinguish.
+- If you tried AI, and it didn't work.
+- The classes are not yet defined, and it's not obvious what you're looking for: you're looking for something new and different.
+- Some classes were not well identified in the AI algorithm that you already tried.
+- No classes are well identified: anomaly detection.
+- You are seeking to classify fewer than a million objects.
+
+
+Three Typical Pathways for Integrating AI and CS
+================================================
+
+There are three typical pathways for combining AI and CS.
+
+1. **Human classifiers prepare data for AI classifiers.** When there is not enough training data for an AI model, human classifiers can perform the initial labeling. This is a common tactic in industry applications. 
+2. **AI classifiers prepare data for humans.** When there is a wealth of data for supervised training of an AI model, but the model persistently struggles to discern certain classes (resulting in false positives and false negatives), it may be appropriate to send this data to CS volunteers for their more nuanced perspective. 
+If there isn't much training data, a clustering method (e.g., k-means clustering or autoencoders) can be used to provide an initial, coarse-grained classification of the data, which can then be sent to CS volunteers.
+3. **Active Learning Hybrid.** Coming soon.
+
+
+
+.. Typical steps for implementing AI with Citizen Science
```

Comment:
Remove all unused, commented-out text.

If this text could be used for future work, create a new Jira ticket for that work, and move this draft text into a file attached to the ticket (or copy-paste it into a comment in the ticket) so it is not lost and can be used in the future. But don't leave it here.